org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -206,7 +267,7 @@ public void testCompletion_3() throws JavaModelException{
 		CompletionItem resolvedItem = server.resolveCompletionItem(ci).join();
 		assertNotNull(resolvedItem.getTextEdit());
 		try {
-			assertTextEdit(5, 4, 6, "put({{key}}, {{value}})", resolvedItem.getTextEdit());
+			assertTextEdit(5, 4, 6, "put(${1:key}, ${2:value})", resolvedItem.getTextEdit());
 		} catch (ComparisonFailure e) {
 			//In case the JDK has no sources
 			assertTextEdit(5, 4, 6, "put({{arg0}}, {{arg1}})", resolvedItem.getTextEdit());
```

Comment:
You need to also make the assertion in case the JDK has no sources in this catch block